content/guides/core-concepts/test-runner.md

```diff
@@ -8,13 +8,15 @@ title: The Test Runner
 
 - The names and purposes of the visual parts of the Cypress Test Runner
```

Comment:
This obv isn't critical but _names & purposes_ might even be correct but reads a little funny to me. Would _names and purpose_ cover all bases in the content?

Reply:
I don't think "purpose" would be correct here, since each visual part has a different purpose, so it would be plural.